src/actions/localization-actions.js

```diff
@@ -50,12 +51,16 @@ export const getLanguageDataAction = ({ language }) => {
 
 export const getLanguageAction = () => {
     return async (dispatch) => {
-        const language = await getLanguage();
-
-        return dispatch({
-            payload: { language },
-            type: "GET_LANGUAGE"
-        });
+        try {
```

Comment:
Please avoid using try catch here. Failures in these functions shoudl not be caught and ignored, but rather propagated to to main error handler, where they are raised as CriticalErrors and stopping the installation.